tests/test-stm-pin-parsing.stanza

```diff
@@ -0,0 +1,481 @@
+    "PE11"                         => "PE[11]",
+    "PE12"                         => "PE[12]",
+    "PE13"                         => "PE[13]",
+    "PE14"                         => "PE[14]",
+    "PE15"                         => "PE[15]",
+    "PE2"                          => "PE[2]",
+    "PE3"                          => "PE[3]",
+    "PE4"                          => "PE[4]",
+    "PE5"                          => "PE[5]",
+    "PE6"                          => "PE[6]",
+    "PE6-WKUP3"                    => "PE[6]",
+    "PE7"                          => "PE[7]",
+    "PE8"                          => "PE[8]",
+    "PE9"                          => "PE[9]",
+    "PF0 / OSC_IN"                 => "PF[0]",
+    "PF0"                          => "PF[0]",
+    "PF0-OSC_IN (PF0)"             => "PF[0]",
+    "PF0-OSC_IN"                   => "PF[0]",
+    "PF1 / OSC_OUT"                => "PF[1]",
+    "PF1"                          => "PF[1]",
+    "PF1-OSC_OUT (PF1)"            => "PF[1]",
+    "PF1-OSC_OUT"                  => "PF[1]",
+    "PF10"                         => "PF[10]",
+    "PF11"                         => "PF[11]",
+    "PF11BOOT0"                    => "PF[11]",
+    "PF12"                         => "PF[12]",
+    "PF13"                         => "PF[13]",
+    "PF14"                         => "PF[14]",
+    "PF15"                         => "PF[15]",
+    "PF2 - NRST"                   => "PF[2]",
+    "PF2"                          => "PF[2]",
+    "PF2-NRST"                     => "PF[2]",
+    "PF3"                          => "PF[3]",
+    "PF4"                          => "PF[4]",
+    "PF5"                          => "PF[5]",
+    "PF6"                          => "PF[6]",
+    "PF7"                          => "PF[7]",
+    "PF8"                          => "PF[8]",
+    "PF9"                          => "PF[9]",
+    "PG0"                          => "PG[0]",
+    "PG1"                          => "PG[1]",
+    "PG10"                         => "PG[10]",
+    "PG10-NRST"                    => "PG[10]",
+    "PG11"                         => "PG[11]",
+    "PG12"                         => "PG[12]",
+    "PG13"                         => "PG[13]",
+    "PG14"                         => "PG[14]",
+    "PG15"                         => "PG[15]",
+    "PG2"                          => "PG[2]",
+    "PG3"                          => "PG[3]",
+    "PG4"                          => "PG[4]",
+    "PG5"                          => "PG[5]",
+    "PG6"                          => "PG[6]",
+    "PG7"                          => "PG[7]",
+    "PG8"                          => "PG[8]",
+    "PG9"                          => "PG[9]",
+    "PH0 - OSC_IN"                 => "PH[0]",
+    "PH0"                          => "PH[0]",
+    "PH0-OSC_IN (PH0)"             => "PH[0]",
+    "PH0-OSC_IN"                   => "PH[0]",
+    "PH0/OSC_IN"                   => "PH[0]",
+    "PH1 - OSC_OUT"                => "PH[1]",
```

Comment:
Is it the expected result? `"PH1 - OSC_OUT"=> "PH[1]"`

Reply:
Yes. It is. They all entered by humans and they all map to PH[0]